src/components/pagination/index.md.njk

```diff
@@ -0,0 +1,118 @@
+
+### For pages in a sequence
+
+For pages the user would usually navigate page by page, only show the previous and next links. Stack them  vertically, like this:
+
+{{ example({group: "components", item: "pagination", example: "labels", html: true, nunjucks: true }) }}
+
+#### Add link labels to describe pages
+
+You can use the link labels to give more context.
+
+{{ example({group: "components", item: "pagination", example: "labels-2", html: true, nunjucks: true }) }}
+
+### For pages in a list
+
+For pages where users might need to jump back and forth, add numbered page links, as well as the previous and next links, like this:
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true, titleSuffix: "second" }) }}
+
+Show the page number in the title of the page so the user can see it in browser tabs and bookmarks. For example 'Search results (page 1 of 4)'.
+
+Do not show pagination if there's only one page of content.
+
+Redirect users to the first page if they enter a URL of a page that no longer exists.
+
+#### Previous and next links
+
+Previous and next links are labelled 'Previous page' and 'Next page'. The word
+'page' can be visually hidden to conserve horizontal space.
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true, titleSuffix: "third" }) }}
+
+#### Customise link titles
+
+You can use custom text for previous and next links, if they're more appropiate and help add context.
+
+{{ example({group: "components", item: "pagination", example: "custom-link-text", html: true, nunjucks: true }) }}
+
+#### First and last pages
+
+Do not show the previous page link on the first page — and do not show the next page link on the last page.
+
+{{ example({group: "components", item: "pagination", example: "first-page", html: true, nunjucks: true }) }}
+
+{{ example({group: "components", item: "pagination", example: "last-page", html: true, nunjucks: true }) }}
+
+### For large numbers of pages
+
+Show an appropriate number of pages to fit the horizontal space available.
+
+{{ example({group: "components", item: "pagination", example: "ellipses", html: true, nunjucks: true }) }}
+
+For smaller screens, show page numbers for:
+- the current page
+- previous and next pages
+- first and last pages
+
+For larger screens, show page numbers for:
+- the current page
+- at least one page immediately before and after the current page 
+- first and last pages
+
+Use ellipses (…) to replace any skipped pages.
+
+For example:
+- **[1]** 2 … 100
+- 1 **[2]** 3 … 100
+- 1 2 **[3]** 4 … 100
+- 1 2 3 **[4]** 5 … 100
+- 1 … 4 **[5]** 6 … 100
+- 1 … 97 **[98]** 99 100
+- 1 … 98 **[99]** 100
+- 1 … 99 **[100]**
+
+### Filtering and sorting lists
+
+If the user filters or sorts the list, apply this to the whole list (not just the current page) and redirect them back to the first page of the new results.
```

Comment:
Could break up this sentence, if that doesn't make it unclear that they do these 2 things (applying filter or sort, and then redirecting) at the same time.

```suggestion
If the user filters or sorts the list, apply their choice to the whole list, and not just the current page. Also redirect them back to the first page of the new results.
```